### DISCUSSION TITLE
Making acc % appear on screen

### DISCUSSION THREAD

**Discussion by AdiQVR** [kinsi55/CS_BeatSaber_Camera2#124] [Q&A]
I've recently seen a bunch of people that have somehow put their accuracy percentage on the screen. Is that with this mod and if so how do i do it too?
![Screenshot 2024-11-21 145929](https://github.com/user-attachments/assets/1145f115-935c-464d-acc6-dcf9a9f63855)

**Comment by kinsi55** [comment]
Yeah you can put a camera up to the score, give it a short render distance and make it transparent
